packages/from-to-location-picker/mocks/trimet-mock.css

```diff
@@ -0,0 +1,22 @@
+.trimet-ambient {
```

Comment:
We should determine if there is a different way to apply different themes and styling. If so, this file might not be needed.

Reply:
This was resolved last week with the decision to use rebass.js (separate PR(s)).

Reply:
Hey Binh,

Cole and I just talked, and he also mentioned that the .css file is a bit funky in how styled-components work.  Rather than doing things in a .css way with class names, he pointed out how the Pulse-Dot is done, and how one can override a default setting: https://github.com/opentripplanner/otp-ui/blob/master/packages/pulse-dot/src/index.js#L4  

Anyway, I'm going to accept the PR, and note that we'll work these things out later.